package.json

```diff
@@ -31,5 +31,17 @@
         "eslint-config-airbnb-base": "^12.1.0",
         "eslint-plugin-import": "^2.9.0",
         "jest": "^22.4.2"
+    },
+    "resolutions": {
+        "mem": "^5.1.1",
+        "lodash": "^4.17.15",
+        "stringstream": "^1.0.0",
+        "js-yaml": "^3.13.1",
```

Comment:
This might be OK (we're increasing from `3.7.0`).

Chains:
* `jest-cli` -> `istanbul-api` -> `js-yaml`
* `eslint` -> `js-yaml`